src/main/resources/lang/default.lang

```diff
@@ -1231,16 +1231,16 @@ entities:
 		pattern: interaction(|1¦s)
 	display:
 		name: display¦s
-		pattern: display(|1¦s)
+		pattern: [a] display(|1¦s) [entity]
 	block display:
 		name: block display¦s
-		pattern: block display(|1¦s)
+		pattern: [a] block display(|1¦s) [entity] [of %-blockdata%]
 	item display:
 		name: item display¦s
-		pattern: item display(|1¦s)
+		pattern: item display(|1¦s) [entity] [of %-itemtype%]
```

Comment:
any reason this doesn't have `[an]`?

Reply:
Because EntityData has `@an`

Reply:
So why do the other two have [a]? I'm confused why some would need an article and some wouldn't.

Reply:
Because existing entity data follow